recipes/suitesparse/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+    cp -f "${RECIPE_DIR}/SuiteSparse_config_Mac.mk" SuiteSparse_config/SuiteSparse_config.mk
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+    cp -f "${RECIPE_DIR}/SuiteSparse_config_linux.mk" SuiteSparse_config/SuiteSparse_config.mk
+fi
+
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+
+export INSTALL_LIB="${PREFIX}/lib"
+export INSTALL_INCLUDE="${PREFIX}/include"
+
+export BLAS="-lopenblas"
+export LAPACK="-lopenblas"
```

Comment:
Though enabling TBB sounds fine and we have that at conda-forge already. I suppose we should also package METIS, but I'm not sure if it is worth the effort as it is packaged with SuiteSparse in 4.5.3.